_posts/2015-03-19-ninja-berries.md

```diff
@@ -0,0 +1,108 @@
+
+I think it takes some trouble, case by case, to argue how learning this or that would benefit a company's immediate concerns.  And as the general case goes, it seems totally impossible to give anything more than generalisations like "learning is good".  Solita has chosen to trust its employees and let them choose.  This gives a borderline between personal and shared different to the traditional solution, with totally new pros and cons.
+
+An employee then is expected to choose those topics that will benefit their competence and work the most.  They may not succeed always, but this is how the market works.  (Pardon me for watering down the traditional [rational action theory](http://en.wikipedia.org/wiki/Rational_choice_theory).)
+
+Now looping back to Hunt's & Thomas' *Diversify* principle, it may benefit me personally in the long run if I learn something quite unrelated to my current tasks, cutting down the risks of my other investments. But then again productivity in my current main responsibilities may often feel more desirable.  In my personal experience, a choice between interesting new topics and established topics is very hard.  It can feel a little selfish to look at something interesting if I could be doing concrete and useful work with that time.
+
+But then again my professional competence and problem solving skills will certainly benefit from an occasional break.
+
+It seems that at Solita the benefits of sharing are estimated higher than the opportunity cost that goes with it.
+
+![drops](/img/ninja-berries/drops.png)
+
+##Conclusion
+
+My nights have been full of thrills lately.  Both of our youngest, my four year old son with his toddler sister, got sick – yielding a nightly vomit per an hour or so.  This has been going on for the last 48 hours.  Now you might think that *ninja berries* refers to those strange orange–white clumps commonly found in that product, whatever!
+
+Going through these personal headaches, it is super great that I can do my own prioritisation at work time, especially on a zombie day like this when logical thinking seems a no–can–do, write a post or take some less important and lower priorised tasks.  I'm not into learning today, being as tired as I am.
+
+This relative freedom of decision was trumpeted in the past by Agile, Scrum, and other production methodologies.  Their common premise is to avoid micromanagement and let the "resource" itself decide its next step.  (Let's provide the cage, management said...)  This same idea, of individual choice within a target driven environment, is transported to the realm of learning here, and it seems to be working fairly well at Solita from what I can judge.
+
+Here are the symptoms:
+
+- an increasing number of professionals kick up short educational meetings, "info strikes"
```

Comment:
"Info strikes" could be replaced with a better term. Can't figure one out though.